src/uu/cut/src/whitespace_searcher.rs

```diff
@@ -0,0 +1,99 @@
+// This file is part of the uutils coreutils package.
+//
+// (c) Rolf Morel <rolfmorel@gmail.com>
```

Comment:
You're not Rolf Morel are you? I don't think we need this line.

Reply:
done